### DISCUSSION TITLE
How to change dark mode as default in mobile environment?

### DISCUSSION THREAD

**Discussion by SHC02** [alshedivat/al-folio#670] [Q&A] [answered]
Hello. First of all, much appreciate for great template. I changed the default page as dark mode in desktop, but it did not apply in mobile. What file should I modify to change the mobile environment? 
Thank you.

**Comment by rohandebsarkar** [comment, marked as answer]
Change the following line
https://github.com/alshedivat/al-folio/blob/f7274c0dab0aed6b0b1c0d7534779546478d1910/assets/js/theme.js#L55
to:
`    if ((userPref && userPref('(prefers-color-scheme: dark)').matches) || userPref('(max-width: 768px)').matches) {`

This will set a default dark theme for mobile and tablet devices.

**Reply by SHC02** [reply]
Thank you so much!!

**Reply by SHC02** [reply]
I'm sorry. It did not work actually.

**Reply by rohandebsarkar** [reply]
There actually was a minor bug. This will be fixed in PR #697. Then this should work.

Another small adjustment, the new line should be:

`if (userPref && (userPref('(prefers-color-scheme: dark)').matches || userPref('(max-width: 768px)').matches)) {`

**Comment by niko-zvt** [comment]
@SHC02 
Hello!
How did you set the default dark theme for desktops?

**Reply by adityajain93** [reply]
if (userPref && (userPref('(prefers-color-scheme: dark)').matches || userPref('(max-width: 768px)').matches || true)) {
